api-admin/openapi.yaml

```
@@ -24,6 +24,19 @@ paths:
         500:
           description: Something goes wrong
 
+  "/users.get":
```

```suggestion
  "/applications/users.list":
```

кажется и остальные нужно в users переименовать